libs/coin-framework/package.json

```diff
@@ -71,10 +61,18 @@
       "require": "./lib/nft/*.js",
       "default": "./lib-es/nft/*.js"
     },
+    "./serialization": {
+      "require": "./lib/serialization/index.js",
```

Comment:
Never understood why it was required to have a `lib` and `lib-es` do know why ? It feels like it's more complicated than it should be

Reply:
It depends if you need `commonjs` or `esm` module.